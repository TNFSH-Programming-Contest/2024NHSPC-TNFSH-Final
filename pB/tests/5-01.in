61 10
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
99622 21 2 143 9 6400701 10918491 592 541345 209379
6 21 2 10 6 28 5 27 10 17 2 8 4
6 6 4 7 10 5 2
69
0 
0 
49
8 44 6 51 5 16 9 8 4 27 8 58 8 5 10 13 1
5 5 8 7 2 9
28
8 59 8 55 4 54 4 21 2 2 2 50 9 18 10 20 7
3 5 10 4
40
7 29 1 41 5 30 7 38 7 49 10 8 9 13 1
8 8 4 2 5 9 10 1 7
48
7 14 7 54 9 34 5 49 9 2 7 21 8 28 8
1 1
67
1 41 8
0 
29
9 30 5 15 5 36 4 50 10 32 3 23 9 52 2 26 4 39 6
5 9 7 1 5 10
63
4 16 10 9 8 51 10 53 7
7 10 5 7 9 3 4 6
57
8 11 4 12 1 1 6 22 4 48 6 5 1 43 4 59 8
10 6 2 5 1 7 9 8 4 3 10
38
1 7 3
1 10
84
3 16 7 35 4 7 10
9 6 9 10 2 8 3 4 1 5
3
5 10 9 9 7 58 4 13 10 34 2
9 9 10 2 7 8 1 4 5 3
78
6 42 5 17 8 25 7 57 9 8 10 30 8
1 3
96
1 4 9
2 7 10
59
10 18 5 45 7 46 2 34 8 5 9 25 1 3 10 48 5 20 8 7 5
9 6 3 7 10 2 9 8 4 5
66
0 
10 5 4 8 1 7 6 2 10 3 9
11
7 61 9 23 6 14 2 50 2 11 5 27 3 38 3
6 5 2 4 3 1 10
55
10 1 3 20 9 34 4 12 5 47 5 18 4 32 4 40 8 30 9 27 8
1 4
31
3 35 7 23 10 12 9
10 6 10 8 7 5 4 9 2 1 3
49
0 
3 3 5 9
30
4 11 4 56 7 7 3 50 7
5 6 10 9 1 3
5
2 22 9 44 2
6 4 8 1 9 10 7
41
2 22 2 56 2
5 5 8 1 9 7
55
9 58 10 18 6 23 3 24 1 41 4 13 8 57 5 38 10 22 8
5 1 4 8 3 2
43
9 47 10 22 6 30 6 44 8 58 9 42 9 38 8 37 1 32 6
6 2 1 9 10 8 6
46
9 35 5 28 5 3 7 8 7 45 8 10 5 54 7 9 4 5 10
0 
52
0 
5 6 1 9 10 4
30
6 50 3 12 3 46 10 41 2 10 2 61 7
2 9 6
75
1 45 3
2 1 6
83
6 9 10 57 7 10 5 29 7 20 9 22 3
10 10 7 3 9 8 2 5 1 4 6
38
10 20 8 57 3 24 9 56 7 17 3 36 6 61 2 46 7 59 9 19 8
3 3 8 6
84
4 41 5 2 7 29 8 43 7
0 
98
0 
10 10 7 6 8 2 3 4 1 5 9
57
8 46 3 15 8 59 2 49 8 35 7 34 5 19 6 36 6
1 6
31
7 17 7 23 10 45 8 32 5 28 2 36 9 54 10
1 3
36
1 16 1
2 8 2
12
1 13 10
6 6 4 2 9 1 3
83
3 10 7 43 6 12 10
8 6 3 10 4 1 8 9 5
27
8 54 9 13 7 6 6 24 8 25 4 61 10 27 8 43 5
3 7 1 3
90
3 59 10 35 9 57 7
8 8 2 9 7 5 6 10 1
51
7 24 8 30 6 21 6 51 3 1 1 31 10 34 6
5 8 10 1 4 7
34
9 19 1 35 4 16 1 26 8 24 2 50 5 9 6 30 3 23 8
3 4 8 2
2
8 7 5 56 6 10 8 52 8 5 2 32 6 34 3 19 8
0 
50
8 27 10 3 3 28 9 10 10 33 9 2 3 9 7 52 10
9 10 8 4 7 3 6 5 2 1
16
1 22 2
8 5 6 9 1 4 8 10 2
44
3 45 6 17 5 10 3
2 9 7
11
5 42 8 29 4 45 8 46 4 56 2
4 9 2 4 10
58
10 21 10 47 9 54 4 57 7 3 9 2 5 45 3 24 4 18 3 36 1
1 8
57
6 39 1 16 1 11 5 54 10 7 5 50 6
9 9 3 5 4 10 2 1 6 7
86
7 36 2 44 1 57 8 7 8 42 4 33 6 27 7
6 9 2 5 6 1 4
87
10 61 5 54 3 55 10 45 8 51 2 27 7 3 10 25 5 19 6 57 10
3 10 3 2
60
9 51 5 5 8 42 8 1 10 34 9 32 6 20 10 27 3 7 3
2 8 7
60
3 56 1 23 1 27 9
4 4 2 5 1
1
9 46 10 34 8 57 10 42 5 40 6 44 6 59 7 27 5 38 5
2 1 5
61
8 39 9 20 7 58 4 48 3 33 4 21 5 8 7 54 3
7 3 6 4 1 9 7 2
11
2 1 9 41 1
1 3
16
7 38 6 4 9 3 1 60 5 5 5 54 8 8 4
8 2 3 6 9 4 5 10 8
8
6 53 3 12 6 48 7 32 8 25 4 6 8
8 3 2 7 5 6 4 10 1
91
1 16 9
1 9
18
10 1 9 42 9 8 6 30 7 16 8 54 9 48 10 45 9 20 3 39 2
6 10 7 8 6 1 4
74
