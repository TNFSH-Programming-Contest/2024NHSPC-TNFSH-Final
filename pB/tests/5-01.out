88
-1
-1
663
-1
75
6400754
-1
-1
-1
11559559
-1
-1
-1
-1
56
-1
-1
-1
11018319
541427
-1
18070595
-1
-1
11018337
-1
-1
-1
-1
7251868
-1
-1
-1
-1
17960823
-1
-1
-1
-1
99845
-1
-1
-1
-1
17629017
18170311
-1
17419719
-1
17628415
-1
11228632
-1
-1
17961026
541421
-1
-1
11670080
18169608
